posthog/batch_exports/sql.py

```diff
@@ -1,5 +1,6 @@
 from django.conf import settings
 
+# TODO: this can now be removed
```

Comment:
Yes, but carefully: These queries are referenced (imported) in historical migrations. Those migrations would need to be updated if this is removed.